molgenis-data-annotators/pom.xml

```diff
@@ -86,45 +86,6 @@
         </dependency>
     </dependencies>
 
-    <profiles>
-        <!-- create runnable jar -->
-        <profile>
-            <id>create-delivery</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <artifactId>maven-assembly-plugin</artifactId>
-                        <configuration>
-                            <archive>
-                                <manifest>
-                                    <mainClass>org.molgenis.data.annotation.cmd.CmdLineAnnotator</mainClass>
-                                    <addDefaultImplementationEntries>
-                                        true
-                                    </addDefaultImplementationEntries>
-                                </manifest>
-                            </archive>
-                            <descriptorRefs>
-                                <descriptorRef>jar-with-dependencies</descriptorRef>
-                            </descriptorRefs>
-                            <finalName>CmdLineAnnotator</finalName>
-                            <appendAssemblyId>false</appendAssemblyId>
-                            <skipAssembly>${skip.delivery}</skipAssembly>
-                        </configuration>
-                        <executions>
-                            <execution>
-                                <id>make-assembly</id> <!-- this is used for inheritance merges -->
-                                <phase>package</phase> <!-- bind to the packaging phase -->
-                                <goals>
-                                    <goal>single</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
-
```

Comment:
this is fixed, however the line note is not on the line of the problem, so github keeps showing this comment.